internal/pkg/template/workload.go

```diff
@@ -522,11 +511,24 @@ func (cfg *ALBListener) RulePaths() []string {
 	return rulePaths
 }
 
-// ServiceConnect holds configuration for ECS Service Connect.
-type ServiceConnect struct {
+// ServiceConnectServer defines the container name and port which a service routes Service Connect through.
+type ServiceConnectServer struct {
+	Name  string
+	Port  string
 	Alias *string
```

Comment:
I thought that `ServiceConnectBoolOrArgs` could have the bool set to `true` in which case we'd want to expose a service connect server, but there wouldn't be an alias? This would make sense as a case for the string to be `nil`. Am I mistaken here?

Reply:
If we make it a `string`, can we not set the field (`nil`) if it is an empty string?

Reply:
Can you elaborate? What's "it" that may be an empty string in this case? Are you saying that we should make Alias of type `string` and set it equal to `"nil"` in the case it's nil? That doesn't make sense so I'm clearly misunderstanding this.

Reply:
I meant we can consolidate empty string and nil cases so that we use `Alias string` instead, but we can keep as is becuase it's not introduced by this PR